lib/loader/index.js

```diff
@@ -77,6 +113,24 @@ module.exports = {
 
     validateArguments(url, options, callback);
 
+    if (options.namespace) {
+      namespace = namespacer.namespace(options.namespace);
```

Comment:
Better to get a static reference to the namespace early than to run through the function each time. It's a small thing here, since this won't be called frequently. Still.